esx_service/cli/vmdkops_admin.py

```diff
@@ -324,7 +430,7 @@ def make_list_of_values(allowed):
     Take a list of allowed values for an option and return a function that can be
     used to typecheck a string of given values and ensure they match the allowed
     values.  This is required to support options that take comma seperated lists
-    such as --rights in 'role set --rights=create,delete,mount'
+    such as --rights in 'tenant set --rights=create,delete,mount'
```

Comment:
Reword this to mention --add-rights or --rm-rights for tenent access set command.